crates/core/tedge_agent/src/tedge_to_te_converter/converter.rs

```diff
@@ -111,6 +112,7 @@ impl TedgetoTeConverter {
             _ => return vec![],
         };
         message.topic = topic;
+        let message  = message.with_retain();
```

Comment:
To be consistent with the previous line:

```suggestion
       message.retain = true;
```